tests/phpunit/tests/modules/test-wpml_elementor-multiple_gallery.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace WPML\PB\Elementor\Modules;
+
+/**
+ * @group modules
+ * @group wpmlcore-7223
+ */
+class TestMultipleGallery extends \OTGS_TestCase {
```

Comment:
It's funny, you used the right class name for the tests :)

Reply:
File is renamed now as you suggested.